docs/api/ShallowWrapper/prop.md

```diff
@@ -2,24 +2,21 @@
 
 Returns the prop value for the node of the current wrapper with the provided key.
 
-NOTE: can only be called on a wrapper of a single node.
+NOTE: can only be called on a wrapper of a single node. Cannot be called on the root wrapper because that represents what the component renders, not the component itself.
```

Comment:
I'm confused. In the issue (linked above) I demonstrated that this is not the case with a test. What am I missing?